Libraries/LibWeb/Layout/SVGFormattingContext.cpp

```diff
@@ -28,6 +28,8 @@
 #include <LibWeb/SVG/SVGSymbolElement.h>
 #include <LibWeb/SVG/SVGUseElement.h>
 
+#include <LibWeb/Layout/Viewport.h>
```

Comment:
Please put this sorted with the others above. (No newline gap.)

Reply:
In the first commit, this include is still out of order. The change to sort the include should be fixup-squashed into the commit that introduced the include.